stories/tour-tooltips.stories.js

```diff
@@ -395,11 +400,12 @@ storiesOf( 'Global', module ).add( 'TourTooltips', () => {
 			title: 'Generic step title for the fourth step',
 			content: (
 				<div>
-					This is the fourth step with an external link that should go
-					to Google,{ ' ' }
-					<Link href="http://google.com" external>
-						link.
-					</Link>
+					{ createInterpolateElement(
+						'This is the fourth step with an external link that should go to Google, <a>link.</a>',
```

Comment:
Not a big deal, but should the dot be inside the link? I think ` <a>link</a>.` is better than ` <a>link.</a>`.